third_party/boringssl/src/crypto/bio/file.c

```diff
@@ -81,12 +81,23 @@
 #define BIO_FLAGS_UPLINK 0
 #endif
 
-#if !defined(OPENSSL_SYS_STARBOARD)
+#if defined(OPENSSL_SYS_STARBOARD)
```

Comment:
Would it be easier to read to combine these #if checks for OPENSSL_SYS_STARBOARD?

Reply:
It could be easier to read, but it also breaks the order of includes per go/cstyle. I personally don't have a preference, lmk what you think